### DISCUSSION TITLE
GRBL and Monport board

### DISCUSSION THREAD

**Discussion by sphericos** [meerk40t/meerk40t#2493] [General]
I recently bought a copy of the [Monport ](https://monportlaser.com/collections/motherboard/products/lightburn-compatible-grbl-motherboard-upgrade-for-desktop-40w-co2-laser-engraver) GRBL board for my K40. I was looking to try and improve the speed and give lightburn a try. Whilst lightburn seems to work fine I am not sure I can tolerate the interface. I have used meerk40t for quite a while and have found it quite simple to run without having to constantly fiddle with settings. However it does not seem to work properly with this board and I wondered if anyone else had got it working properly. It detects the device but the best I have managed to do is to get it to jog in one axis.

**Comment by jpirnay** [comment]
From the screenshots I saw it claims to be a GRBL 1.1f compatible board?! It could be one of several things:
a) incompatible speed seetings for the USB connection (not very likely, still worth experimenting a bit with different settings)
b) MeerK40t not properly interacting ie unaware of the controllers specific feedbacks
c) ...
So you might want to experiment a bit with the connection settings. Additionally it would be helpful if you would open the contoller window and enter a gcode command like ``G0 X50 Y50 Z0``:
![grafik](https://github.com/meerk40t/meerk40t/assets/2670784/6eb615fe-f018-4f39-ae83-ac7a4a277841)
What's the output generated? And does the laser move to (50, 50)?

**Reply by sphericos** [reply]
Thanks for the reply it looks like user error on my part. The x axis was reversed and it seems to have been going into alarm. It seems to all be working now but I am still testing.

Cheers
